src/styles/layout/common-two-thirds-one-third/index.njk

```diff
@@ -15,12 +15,12 @@ ignore_in_sitemap: true
 
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
-        <h1 class="govuk-heading-xl">Two-thirds column</h1>
+        <h1 class="govuk-heading-l">Two-thirds column</h1>
```

Comment:
What's with the sizing changes here? Are they related?

Reply:
No they are not related. It should have just been the heading levels that changed. good spot